Makefile

```diff
@@ -98,6 +98,7 @@ endif
 .k0sbuild.docker-image.k0s: build/Dockerfile
 	docker build --rm \
 		--build-arg BUILDIMAGE=golang:$(go_version)-alpine \
+		-f build/Dockerfile \
```

Comment:
Wondering why that's necessary...

Reply:
I don't think its necessary, but it makes it a bit more readable. more visible what is going on.